source/types.ts

```diff
@@ -173,6 +173,8 @@ export type Store = {
 	localKeys?: boolean
 }
 
+export type DraftHeadersVersion = 'draft-6' | 'draft-7'
```

Comment:
Maybe this should be an enum? I'm not sure. At any rate, this seems to work.

Reply:
This is perfect